qa/L0_backend_python/test.sh

```diff
@@ -153,6 +153,8 @@ else
   # GPU tensor tests are disabled on jetson
 fi
 
+pip3 install pytest
```

Comment:
Now that we are using pytest in all the tests it might be worth installing it in the QA container.

Reply:
I think Pytest is already there with our default Python version. Installing Pytest in the test is just for the different python environments.